packages/dataviews/src/constants.ts

```diff
@@ -3,49 +3,6 @@
  */
 import { __ } from '@wordpress/i18n';
 
-// Filter operators.
```

Comment:
I wrote a big comment about how we can rely more on the type system and make a lot of these constants irrelevant without sacrificing clarity or readability (in my opinion).

Instead of writing the comment, I pushed a change that implements that idea. Please feel free to keep it or revert it, but it's a powerful concept and a compelling way to rely on the information the type system provides.